apps/server/src/modules/tool/service/tool-validation.service.ts

```diff
@@ -90,6 +90,10 @@ export class ToolValidationService {
 				try {
 					// eslint-disable-next-line no-new
 					new RegExp(param.regex);
+					if (param.default) {
+						// eslint-disable-next-line no-new
+						new RegExp(param.default);
```

Comment:
This does not work. If you want to validate the regex you need to do something like this: param.default.match(new RegExp(param.regex))